src/main/java/com/soneso/stellarmnemonics/mnemonic/Mnemonic.java

```diff
@@ -71,4 +71,30 @@
             throw new MnemonicException("Fatal error when generating seed from mnemonic!");
         }
     }
+
+    public static byte[] createSeed(char[] mnemonic, char[] passphrase, int apiLevel) throws MnemonicException {
+
+        char[] saltChars = new char[]{'m', 'n', 'e', 'm', 'o', 'n', 'i', 'c'};
+        if (passphrase != null) {
+            saltChars = PrimitiveUtil.concatCharArrays(saltChars, passphrase);
+        }
+        byte[] salt = PrimitiveUtil.toBytes(saltChars);
+
+        try {
+            KeySpec ks = new PBEKeySpec(mnemonic, salt, 2048, 512);
+            SecretKeyFactory skf = null;
+            if (apiLevel >= 26) {
+                skf = SecretKeyFactory.getInstance("PBKDF2WithHmacSHA512");
+            } else if (apiLevel >= 19) {
+                skf = SecretKeyFactory.getInstance("PBKDF2withHmacSHA1And8BIT");
+            } else if (apiLevel >= 10) {
+                skf = SecretKeyFactory.getInstance("PBKDF2withHmacSHA1");
+            } else {
+                skf = SecretKeyFactory.getInstance("PBEwithSHA1ANDDES");
```

Comment:
I randomly picked an algorithm for level 1+, so please use another one if need be.